Formula/qt5.rb

```diff
@@ -90,12 +100,32 @@ def install
       -qt-pcre
       -nomake tests
       -no-rpath
+      -pch
+      -system-proxies
+      -optimized-tools
     ]
 
     args << "-nomake" << "examples" if build.without? "examples"
 
-    args << "-plugin-sql-mysql" if build.with? "mysql"
-    args << "-plugin-sql-psql" if build.with? "postgresql"
+    if build.without? "plugins"
+      use_plugin = "qt"
+      args << "-qt-sql-sqlite"
+      inreplace "qtbase/configure" do |s|
+        s.gsub! "CFG_JPEG=plugin", "CFG_JPEG=yes"
+        s.gsub! "CFG_GIF=plugin", "CFG_GIF=yes"
```

Comment:
it's not a bug, Like I said, there are options that build sql driver as plugin or built into QtSql. But for gif/jpeg, there is no such switch, the default behavior is just build as plugin when build shared link qt libs, or built into qt gui when build static link qt.

and this gif/jpeg plugin is purely my personal favorite.

as to the system-proxy, I doubt it will break abi compatibly.  also, If I'm developing a qt based applications, I will go to the system-proxy way.